Flexa-Agents/Fallback-State/README.md

```diff
@@ -0,0 +1,19 @@
+# Fallback State Agent
+
+## Description
+
+This agent detects the contract entering a fallback state
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- FLEXA-4
+  - Fired whenever the contract enters a fallback state
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata fields
+    - blockTimestamp: Timestamp of the block
+    - blockNumber: Block number of the block
```

Comment:
I'll remove metadata since it's not needed for this agent as Lazaro mentioned below. https://github.com/NethermindEth/Forta-Agents/pull/163#discussion_r821126700